test/modelcheck/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```

Comment:
So this PR also updates Gradle version? Is this intended?

Reply:
I want to delete complete folder later since its no longer required. Please ignore for now.